exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,45 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+// function isNegative(num){
+//   if (num < 0){
+//     return true;
+//   } else {
+//     return false;
+//   }
+// }
+
+let isNegative = (num) => num < 0 ? true : false;
```

Comment:
Functions like these could be simplified even further. Whenever you're doing something like `expression ? true : false`, you could instead return the expression directly - as it will already be evaluated to either true or false.